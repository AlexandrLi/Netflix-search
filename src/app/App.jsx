import React from 'react';
import { SearchPage } from '../search-page/SearchPage'
import { FilmInfo } from '../film-info/FilmInfo'
import * as s from './app.css';

export class App extends React.Component {
  render() {
    return (
      <div>
        <FilmInfo />
        <footer>
          netflixroulette
        </footer>
      </div>
    )
  }
}