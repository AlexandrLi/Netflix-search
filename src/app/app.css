footer {
  font-family: 'Work Sans', sans-serif;
  display: flex;
  align-items: center;
  background-color: #414141;
  color: #F55263;
  padding-left: 40px;
  height: 50px;
}

.content {
  height: 500px;
}