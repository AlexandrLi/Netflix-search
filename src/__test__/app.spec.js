import React from "react"
import renderer from "react-test-renderer"
import { shallow } from "enzyme";
import { App } from "../app/App";

describe("App snapshot", () => {
  it("Take App snapshot", () => {
    const renderedSearchPage = renderer.create(<App />).toJSON();
    expect(renderedSearchPage).toMatchSnapshot();
  })
});

describe("Shalow render of App component", () => {
  let component;
  beforeEach(() => {
    component = shallow(<App />)
  });

  it("Component successfully rendered", () => {
    expect(component.length).toEqual(1);
  });
});