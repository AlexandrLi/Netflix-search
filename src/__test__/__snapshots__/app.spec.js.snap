// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot Take App snapshot 1`] = `
<div>
  <footer>
    netflixroulette
  </footer>
</div>
`;
